lib/theme/hs_color.dart

```diff
@@ -26,6 +26,7 @@ class HSColor {
   static const Color statusSuccess = Color(0xFF34C759);
   static const Color statusInformational = Color(0xFF007AFF);
 
+  static const Color neutral9 = Color(0xFF141414);
```

Comment:
please add comment 'new design' so we know what value has been updated with new design